examples/03_connectivity/plot_compare_resting_state_dictlearning_masking.py

```diff
@@ -0,0 +1,112 @@
+"""
+Dictionary Learning with different masking strategies for analysis of
+resting-state fMRI
+==========================================================================
```

Comment:
This is not valid restructured text: the title must be on one line, and the line below must extend as long as this line, maybe longer.